plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -945,29 +941,23 @@ public function get_post_type_counts() {
 	 * @return array
 	 */
 	public function get_active_plugins() {
-		$active_plugins_data = get_transient( 'wc_system_status_active_plugins' );
+		require_once ABSPATH . 'wp-admin/includes/plugin.php';
```

Comment:
Why these changes? They seem unrelated to the issue of the memory limit display, or am I missing something?